packages/app/src/components/Dimensions/DataDimension/SelectedItems.js

```diff
@@ -2,16 +2,14 @@ import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
 import i18n from '@dhis2/d2-i18n';
-import {
-    UnAssignButton,
-    DeselectAllButton,
-    RemoveSelectedItemButton,
-} from './buttons';
+import { Item } from './Item';
+import { UnAssignButton, DeselectAllButton } from './buttons';
 
 import { sGetMetadata } from '../../../reducers/metadata';
+import { toggler } from './toggler';
 import { colors } from '../../../colors';
 
-const style = {
+const styles = {
```

Comment:
Move this style object as well to the new styles/ folder (styles/SelectedItems.styles.js)